specs/kubernetes/exec_shell_inside_pod.yaml

```diff
@@ -0,0 +1,10 @@
+name: Opens a Bash shell into Kubernetes pod
+command: kubectl -n {{namespace}} exec -it {{pod_name}} -- /bin/sh
+arguments:
+  - name: namespace
+    default_value: default
+    description: The namespace of the pod
+  - name: pod_name
+    description: The pod into which to exec
+
+tags: ["kubectl", "exec", "shell", "bash"]
```

Comment:
nit: in general the Warp team recommends reducing the number of tags and to pick ones that are more descriptive. Maybe just `kubectl` here?

Reply:
@radugalbenu Lmk when this is addressed and ready for another pass!

Reply:
@alokedesai I have created a new PR with the suggested changes, I messed up with the branches I used.
New PR here: https://github.com/warpdotdev/workflows/pull/139 . thanks!